Dockerfile.dapper

```diff
@@ -43,7 +43,7 @@ RUN if [ ${ARCH} == "s390x" ]; then \
 ENV GOLANG_ARCH_amd64=amd64 GOLANG_ARCH_arm64=arm64 GOLANG_ARCH_s390x=s390x GOLANG_ARCH=GOLANG_ARCH_${ARCH} \
     GOPATH=/go PATH=/go/bin:/usr/local/go/bin:${PATH} SHELL=/bin/bash
 RUN wget -O - https://storage.googleapis.com/golang/go1.14.15.linux-${!GOLANG_ARCH}.tar.gz | tar -xzf - -C /usr/local && \
-    go get github.com/rancher/trash && go get -u golang.org/x/lint/golint
+    go get github.com/rancher/trash
```

Comment:
Why remote golint?

Reply:
Changed back, and keep it failed when getting `golint`, It should be a go version issue.